### PR TITLE
[ci] Run Rubocop with Ruby 2.5.0

### LINKED ISSUE #4808: [ci] rubocop runs with ruby 2.1 parser
Issue/Feature description
====================
Rubocop runs either with the ``.ruby-version``, ``Gemfile.lock`` or as configuration
```
AllCops:
  TargetRubyVersion: 2.5
```

Neither is set by OBS so rubocop uses 2.1 parser which does raise Lint/Syntax errors which is not possible to disable (e.g. disallows ``<<~`` heredoc).

Setting TargetRubyVersion to 2.5 resulted in basically a violation in almost every file, so I wanted to create an issue first.

Expected result
============
We use ruby parser 2.5 in rubocop

How to Reproduce
===============
1. Add ``<<~`` somewhere in the code and run rubocop
or
1. Add ``TargetRubyVersion: 2.5`` and run rubocop to see all the violations

### PR DESCRIPTION
**Pull request by dmarcoux**
A much less drastic approach would be to disable the cop `Style/FrozenStringLiteralComment`

*Before continuing in fixing offenses, I prefer to get some feedback if they should be corrected (auto-corrected when possible) or disabled*

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.5.0
```
Comment (Ana06) [inline .ruby-version]: why doing it like that instead of setting `TargetRubyVersion: 2.5` in the `.rubocop.yml` file?

File: src/api/Gemfile [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
+
```
Comment (Ana06) [inline src/api/Gemfile]: I wonder why that is not done by default in the autocorrect of the `Style/FrozenStringLiteralComment`.... I'll open an issue in Rubocop :thinking:

File: .ruby-version (hunk 1 above)
Comment (dmarcoux) [reply .ruby-version]: This file is picked up by Rubocop automatically. See the documentation here: https://github.com/bbatsov/rubocop/blob/52d03b4b1d9296441360a4e4bbe0498be1be1141/manual/configuration.md#setting-the-target-ruby-version

**Comment by dmarcoux** [comment]
@Ana06 I realized that there were other remaining Rubocop offenses after creating the PR (and seeing Travis fail). I will tackle the remaining offenses soon.

**Comment by dmarcoux** [comment]
@Ana06 Good point with the todo file. I will do this instead. I'm closing this PR.

File: .ruby-version (hunk 1 above)
Comment (Ana06) [reply .ruby-version]: I know, my question was why doing it this way and not setting the target in the config file... I am actually not sure where it is less likely that we forget to change it in the next update :thinking:

**Pull request closed without merging**